members/templates/members/union_members.html

```diff
@@ -0,0 +1,94 @@
+{% extends 'members/base.html' %}
+
+{% block content %}
+  <div id="union_info">
+      <h1>Foreningen for {{union.name}}</h1>
+      <strong>Adresse: </strong>{{union.address}}
+      <div>
+        <h4>Bestyrelse</h4>
+        <div>
+          <div>
+              <ul>
+                <li>
+                    <strong>Formand:</strong>
+                    <a href="mailto:{{union.chairman.email}}">{{union.chairman.name}}</a>
+                </li>
+                <li>
+                    <strong>Næstformand:</strong>
+                    <a href="mailto:{{union.second_chair.email}}">{{union.second_chair.name}}</a>
+                </li>
+                <li>
+                    <strong>Kasserer:</strong>
+                    <a href="mailto:{{union.cashier.email}}">{{union.cashier.name}}</a>
+                </li>
+                <li>
+                    <strong>Sekretær:</strong>
+                    <a href="mailto:{{union.secratary.email}}">{{union.secretary.name}}</a>
+                </li>
+                <li>
+                    <strong>Foreningsmail:</strong>
+                    <a href="mailto:{{union.union_email}}">{{union.union_email}}</a>
+                </li>
+              </ul>
+          </div>
+          <div>
+            <strong>Menige medlemmer</strong>
+            <div>
+              {% for boardMember in BoardMembers %}
+                <a href="mailto:{{boardMember.email}}">{{boardMember.name}}</a><br>
+              {% endfor %}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div id="union_documents">
+        {% if union.statues %}
+            <a href="{{union.statues}}"><strong>Vedtægter</strong></a>
+        {% endif %}
+        {% if union.meeting_notes %}
+            <a href="{{union.meeting_notes}}"><strong>Referater</strong></a>
+        {% endif %}
+    </div>
+    {% if access %}
+      <div class="tabs reverse">
+        <ul>
+          {% for year in years %}
+            <li>{{year}}</li>
+          {% endfor %}
+        </ul>
+        {% if members.items %}
+          {% for year, memberList in members.items %}
+            <section>
+              <table class="table table-striped">
```

Comment:
Er der en grund til at bruge bootstrap table i stedet for vores egen? 
Og virker den så på mobil?

Reply:
Har vist bare brugt samme table som alle de andre steder jeg kunne finde på det tidspunkt, og det skulle gerne virke på mobil også ja.

Reply:
Tjekkede lige nu da det hele rent faktisk virker, og tabellen fungerer også på mobil (i hvert fald på min iPhone)